docs/source/index.rst

```diff
@@ -49,7 +49,7 @@ Composer is part of the broader Machine Learning community, and we welcome any c
 .. toctree::
    :hidden:
    :maxdepth: 1
-   :caption: Examples
+   :caption: Notebooks
```

Comment:
Even though they are notebooks, would prefer to keep them called "examples" or "tutorials"  -- I think that is the more standard terminology (pytorch calls them tutorials; ptl has tutorials and examples)

Reply:
Changed to Tutorials